acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/ProcessBesuNodeRunner.java

```diff
@@ -278,10 +278,11 @@ public void startNode(final BesuNode node) {
     params.add("--auto-log-bloom-caching-enabled");
     params.add("false");
 
-    String level = System.getProperty("root.log.level");
-    if (level != null) {
-      params.add("--logging=" + level);
-    }
+    // String level = System.getProperty("root.log.level");
+    // if (level != null) {
+    //  params.add("--logging=" + level);
+    // }
+    params.add("--logging=TRACE");
```

Comment:
What's the advantage of doing it this way over, say, changing the `.circleci/config.yaml` to have `TRACE` as the `root.log.level`?

Reply:
Yeah, This PR isn't meant to go in - it was just to be able to trigger CI to determine underlying cause.

Reply:
Ahh, ok